src/main/java/core/basesyntax/sevice/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.sevice.impl;

```

Comment:
The package name `sevice` seems to be a typo. It should be `service` as per the standard Java package naming conventions and to match the project's directory structure.